resources/prometheus/prometheus-rules.yaml

```diff
@@ -454,6 +454,78 @@ spec:
+          record: central:http_incoming_request_duration_histogram_seconds_bucket:rate10m:p90
+
+        # - Queries the current central API latency (GRPC and HTTP) SLI by calculating the ratio of successful
+        #   instances of central:xxx:rate10m:p90 over its total instances for a certain period.
+        # - Note that to get the current SLI with a variable PERIOD, simply run the following query where PERIOD is the desired period in
+        #   promql duration format. This query is useful for dynamically determining an SLI regardless of an SLO.
+        #
+        #   sum_over_time(central:http_incoming_request_duration_histogram_seconds_bucket:rate10m:p90[PERIOD]) / count_over_time(central:http_incoming_request_duration_histogram_seconds_bucket:rate10m:p90[PERIOD])
+        #
+        - expr: |
+            sum_over_time(central:grpc_server_handling_seconds:rate10m:p90[28d]) / count_over_time(central:grpc_server_handling_seconds:rate10m:p90[28d])
+          record: central:grpc_server_handling_seconds:rate10m:p90:sli
+        - expr: |
+            sum_over_time(central:http_incoming_request_duration_histogram_seconds_bucket:rate10m:p90[28d]) / count_over_time(central:http_incoming_request_duration_histogram_seconds_bucket:rate10m:p90[28d])
+          record: central:http_incoming_request_duration_histogram_seconds_bucket:rate10m:p90:sli
+
+        # - Queries the error rate or the ratio of the instances of central:xxx:rate10m:p90
+        #   were equal to 0 over the total instances of central:xxx:rate10m:p90 within a period.
+        - expr: |
+            1 - central:grpc_server_handling_seconds:rate10m:p90:sli
+          record: central:grpc_server_handling_seconds:rate10m:p90:error_rate28d
+        - expr: |
+            1 - central:http_incoming_request_duration_histogram_seconds_bucket:rate10m:p90:sli
+          record: central:http_incoming_request_duration_histogram_seconds_bucket:rate10m:p90:error_rate28d
+
+        # - Queries error rate for a 1h window.
+        - expr: |
+            1 - (sum_over_time(central:grpc_server_handling_seconds:rate10m:p90[1h]) / count_over_time(central:grpc_server_handling_seconds:rate10m:p90[1h]))
+          record: central:grpc_server_handling_seconds:rate10m:p90:error_rate1h
+        - expr: |
+            1 - (sum_over_time(central:http_incoming_request_duration_histogram_seconds_bucket:rate10m:p90[1h]) / count_over_time(central:http_incoming_request_duration_histogram_seconds_bucket:rate10m:p90[1h]))
+          record: central:http_incoming_request_duration_histogram_seconds_bucket:rate10m:p90:error_rate1h
+
+        # - Queries the error budget exhaustion (or consumption) for the whole slo window (28d).
+        - expr: |
+            (1 - central:grpc_server_handling_seconds:rate10m:p90:sli) / 0.01
```

Comment:
Can you define a scalar recording rule for the target (`0.01` / `0.99`)? That allows us to change the value in a single place.